Jarfile.lock

```diff
@@ -1,66 +1,68 @@
 ---
 :runtime:
-- io.dropwizard.metrics:metrics-core:jar:3.1.1
```

Comment:
There shouldn't be any `Jarfile.lock` change for this PR (the `Jarfile` dictates Kill Bill dependencies versions for tests, which haven't changed).